quakemigrate/plot/phase_picks.py

```diff
@@ -72,82 +76,165 @@ def pick_summary(event, station, signal, picks, onsets, ttimes, window):
+    colors = ["#F03B20", "#3182BD"]
+    if len(phases) == 1:
+        if phases[0] == "P":
+            ax_ind = [0, 3]
+            colors = [colors[0]]
+        else:
+            ax_ind = [1, 2, 4]
+            colors = [colors[1]]
+
+    # Loop through all relevant axes
+    for ind in ax_ind:
+        ax = axes[ind]
+        # Plot model picks
+        model_pick = otime + ttimes[0] if ind % 3 == 0 else otime + ttimes[-1]
+        ph = phases[0] if ind % 3 == 0 else phases[-1]
         ax.axvline(model_pick.datetime, alpha=0.9, c="k",
-                   label="Modelled pick time")
+                   label=f"Modelled {ph} arrival")
+        # Plot event origin time if it is on plot:
+        if mpl_times[min_idx] < otime.matplotlib_date:
+            ax.axvline(event.otime.datetime, c="green",
+                       label="Event origin time")
+        # Plot pick windows
+        win = windows[phases[0]] if ind % 3 == 0 else windows[phases[-1]]
+        clr = colors[0] if ind % 3 == 0 else colors[-1]
+        ax.axvspan(mpl_times[win[0]], mpl_times[win[2]], alpha=0.2, color=clr,
+                   label="Picking window")
+        # Set xlim
+        ax.set_xlim(mpl_times[min_idx], mpl_times[max_idx])
 
     # --- Plot picks and summary information ---
-    text = fig.add_subplot(3, 2, 1)
-    text.text(0.5, 0.8, f"Event: {event.uid}\nStation: {station}",
-              ha="center", va="center", fontsize=22, fontweight="bold")
     for i, pick in picks.iterrows():
         # Pick lines
         if pick["Phase"] == "P":
             c1, c2 = "#F03B20", "gray"
         else:
             c1, c2 = "gray", "#3182BD"
         if pick["PickTime"] != -1:
-            for j, ax in enumerate(axes):
-                clr = c1 if j % 3 == 0 else c2
+            for ind in ax_ind:
+                ax = axes[ind]
+                clr = c1 if ind % 3 == 0 else c2
                 _plot_phase_pick(ax, pick, clr)
-
+        # Calculate residual:
+        if pick.PickTime == -1:
+            resid = -1
+        else:
+            resid = pick.PickTime - pick.ModelledTime
         # Summary text
         text.text(0.1+i*0.5, 0.6, f"{pick.Phase} phase", ha="center",
                   va="center", fontsize=20, fontweight="bold")
         pick_info = (f"Pick time: {pick.PickTime}\n"
                      f"Pick error: {pick.PickError:5.3f} s\n"
-                     f"Pick SNR: {pick.SNR:5.3f}")
-        text.text(0.05+i*0.5, 0.45, pick_info, ha="left", va="center",
+                     f"Pick SNR: {pick.SNR:5.3f}\n"
+                     f"Pick residual: {resid:5.3f} s")
+        text.text(0.05+i*0.5, 0.4, pick_info, ha="left", va="center",
                   fontsize=18)
     text.set_axis_off()
 
-    for ax in axes[3:5]:
-        ax.legend()
-    fig.tight_layout(pad=1, w_pad=0)
+    # Add legend
+    for ind in ax_ind:
+        if ind > 2:
+            axes[ind].legend(fontsize=16, loc="upper right")
+
+    fig.tight_layout(pad=4, w_pad=4)
```

Comment:
```suggestion
    fig.tight_layout(pad=1)
```
1 / 0 is consistent with our other plots, but I think `w_pad=1` is necessary to separate the P and S onset y-axes.